.circleci/config.yml

```diff
@@ -64,11 +70,48 @@ jobs:
           path: build/test-results
           working_directory: android
 
+  deploy-android:
+    executor: android-executor
+    steps:
+      - checkout
+      - android/accept-licenses
+      - restore_cache:
+          key: jars-{{ checksum "android/build.gradle" }}
+      - android/restore-build-cache
+      - run:
+          name: Deployment
+          command: |
+            REPO="purchases-hybrid-common"
+            USERNAME="RevenueCat"
+            JDK="oraclejdk8"
+
+            if [ "$CIRCLE_PROJECT_REPONAME" != "$REPO" ]; then
+              echo "Skipping snapshot deployment: wrong repository. Expected '$REPO' but was '$CIRCLE_PROJECT_REPONAME'."
+            elif [ "$CIRCLE_PROJECT_USERNAME" != "$USERNAME" ]; then
+              echo "Skipping snapshot deployment: wrong owner. Expected '$USERNAME' but was '$CIRCLE_PROJECT_USERNAME'."
+            elif [ "$CIRCLE_JDK_VERSION" != "$JDK" ]; then
+              # $CIRCLE_JDK_VERSION must be manually set in circle.yml
+              echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$CIRCLE_JDK_VERSION'."
+            else
+              echo "Deploying snapshot..."
+              ./gradlew androidSourcesJar androidJavadocsJar uploadArchives -P signing.keyId=$GPG_SIGNING_KEY_ID -Psigning.password=$GPG_SIGNING_KEY_PW -Psigning.secretKeyRingFile=./secring.gpg \
+                                      -PSONATYPE_NEXUS_USERNAME=$SONATYPE_NEXUS_USERNAME -PSONATYPE_NEXUS_PASSWORD=$SONATYPE_NEXUS_PASSWORD -PRELEASE_SIGNING_ENABLED=true
```

Comment:
maybe we could break lines in between params?

Reply:
as in 
```sh
./gradlew androidSourcesJar androidJavadocsJar uploadArchives \
    -P signing.keyId=$GPG_SIGNING_KEY_Iassword=$GPG_SIGNING_KEY_PW \
    -Psigning.secretKeyRingFile=./secring.gpg \
    -PSONATYPE_NEXUS_USERNAME=$SONATYPE_NEXUS_USERNAME \
    -PSONATYPE_NEXUS_PASSWORD=$SONATYPE_NEXUS_PASSWORD \
    -PRELEASE_SIGNING_ENABLED=true
```

Reply:
also, is there a typo in the second param? is it supposed to be `-P signing.secretKeyRingFile` instead of `-Psigning.secretKeyRingFile`?

Reply:
That's strange! it's the same in the purchases-android script https://github.com/RevenueCat/purchases-android/blob/develop/.buildscripts/deploy_snapshot.sh

Reply:
just realized that there's another param that has the same weirdness, `signing.password`

Reply:
I doubled checked in other examples and looks like it's `-Psigning` instead of `-P signing` in both cases